src/app/Configuration.js

```diff
@@ -25,6 +25,8 @@ module.exports = class Configuration {
         let applicationExecutableDirectory = path.dirname(electron.app.getPath('exe'));
```

Comment:
Can revert changes in this file since the `cubeloader.html` is now statically applied in `App.js`